### DISCUSSION TITLE
Action Stuck in Scheduled & Running

### DISCUSSION THREAD

**Discussion by mgazzali** [StackStorm/st2#5886] [Operations]
I have installed Stackstorm 3.7 previously using Stackstorm HA Helm Chart

Stackstorm run fine for the first month, however recently there is hoard of automation kicking in and causing bunch of Action stuck in running / scheduled for few days

![Screenshot 2023-02-05 133019](https://user-images.githubusercontent.com/109259265/216803435-b7dc42cb-8729-43df-a2e3-b1715d4ca072.png)

Any recommendation on how to fix this? What is the potential issue that caused this?

**Comment by arm4b** [comment]
Moving this issue to `stackstorm/st2` discussions.

**Comment by arm4b** [comment]
@mgazzali  You likely hit some bottleneck with performance in the cluster due to higher load spike.
If you have metrics around MongoDB, RabbitMQ and stackstorm itself, - those could answer what was the real bottleneck and help to scale you out.

**Reply by acharbha** [reply]
what to check in st2 in this case ?
